src/styles/components/_card-author.scss

```diff
@@ -0,0 +1,13 @@
+@import '../settings/colors';
+
+.w-card-author {
+  text-align: center;
+}
+
+.w-card-author__figure {
+  background-color: $WHITE;
+}
+
+.w-card-author__image {
```

Comment:
add a width: 192px to these. I think that's the highest resolution image we have of folks.